asb-native/src/main/java/org/ballerinax/asb/receiver/MessageReceiver.java

```diff
@@ -0,0 +1,363 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
As we are creating a new file now, we can change the year to 2021

Reply:
Updated with the year 2021.